ui/src/main/java/edu/wpi/grip/ui/WarningAlert.java

```diff
@@ -0,0 +1,69 @@
+package edu.wpi.grip.ui;
+
+import org.pegdown.PegDownProcessor;
+
+import javafx.scene.control.Alert;
+import javafx.scene.effect.BlendMode;
+import javafx.scene.web.WebEngine;
+import javafx.scene.web.WebView;
+import javafx.stage.StageStyle;
+import javafx.stage.Window;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * An alert that warns the user when they try to do something unsupported, such as trying to
+ * generate code with an operation that doesn't support code gen.
+ *
+ * <p>The alert renders the body text as markdown.
+ * </p>
+ */
+public class WarningAlert extends Alert {
+
+  /**
+   * Markdown processor. Converts markdown to HTML.
+   */
+  private static final PegDownProcessor mdProcessor = new PegDownProcessor();
+  private final WebView webView = new WebView();
+  private final WebEngine webEngine = webView.getEngine();
+
+  // Width and height of the alert dialog
+  private static final double WIDTH = 360;
+  private static final double HEIGHT = 180;
```

Comment:
This will be removed once I get the dialogues to properly fit the rendered content